loggy/lib/src/filters/blacklist_filter.dart

```diff
@@ -0,0 +1,11 @@
+part of loggy;
+
+/// Black list filter, opposite of [WhitelistFilter]
+class BlacklistFilter extends WhitelistFilter {
+  BlacklistFilter(List<Type> types) : super(types);
+
+  @override
+  bool shouldLog(LogLevel level, Type type) {
+    return !super.shouldLog(level, type);
```

Comment:
This ties in with my comment for `WhitelistFilter`.

In the example below:
Nothing gets logged.
I would expect the log to be: `👻 08:05:46.351346 INFO Example - abc`

```dart
void main() {
  Loggy.initLoggy(
    logPrinter: const PrettyPrinter(),
    logOptions: const LogOptions(
      LogLevel.all,
    ),
    filters: [
      BlacklistFilter([]),
    ],
  );

  MyExample();
}

class MyExample with ExampleLoggy {
  MyExample(){
    loggy.info("abc");
  }
}
```